cms/templates/widgets/header.html

```diff
@@ -20,23 +20,24 @@ <h1 class="branding"><a href="/">
       % if context_course:
       <%
             course_key = context_course.id
+            url_encoded_course_key = quote(six.text_type(course_key).encode('utf-8'), safe='')
             current_organization = context_course.org
-            index_url = reverse('course_handler', kwargs={'course_key_string': str(course_key)})
-            course_team_url = reverse('course_team_handler', kwargs={'course_key_string': str(course_key)})
-            assets_url = reverse('assets_handler', kwargs={'course_key_string': str(course_key)})
-            textbooks_url = reverse('textbooks_list_handler', kwargs={'course_key_string': str(course_key)})
-            videos_url = reverse('videos_handler', kwargs={'course_key_string': str(course_key)})
-            import_url = reverse('import_handler', kwargs={'course_key_string': str(course_key)})
-            course_info_url = reverse('course_info_handler', kwargs={'course_key_string': str(course_key)})
-            export_url = reverse('export_handler', kwargs={'course_key_string': str(course_key)})
-            settings_url = reverse('settings_handler', kwargs={'course_key_string': str(course_key)})
-            grading_url = reverse('grading_handler', kwargs={'course_key_string': str(course_key)})
-            advanced_settings_url = reverse('advanced_settings_handler', kwargs={'course_key_string': str(course_key)})
-            tabs_url = reverse('tabs_handler', kwargs={'course_key_string': str(course_key)})
+            index_url = reverse('course_handler', kwargs={'course_key_string': six.text_type(course_key)})
```

Comment:
@OmarIthawi We're only running this with Python3 now, right?

Reply:
Yes @johnbaldwin. This is a straight copy from Juniper (which is still not migrated away from Python 2.7), think about it as a raw git merge, so we don't really control the changes here much. 

The reason it had the `str` because we copied a lot of changes from Koa which was premature and caused styles to break.

I've added the "Changes" section to list all changes that was intended to be done.

Reply:
Thanks.  I've skimmed over the resources, but not in any great detail. You know this all better than me and as long as you're ok with leaving the six handling in there, I'm not going to make an issue out of it. Py2/3 compatibility is probably the most minor of tech debt in the platform